etc/profile-a-l/freetube.profile

```diff
@@ -6,26 +6,22 @@ include freetube.local
 # Persistent global definitions
 include globals.local
 
+# Disabled until someone reported positive feedback
```

Comment:
I have been running freetube without issues having these in my freetube.local:

include whitelist-common.inc
include whitelist-runuser-common.inc
include whitelist-usr-share-common.inc
include whitelist-var-common.inc

nou2f
novideo

The nou2f is untested due to not having such hardware.

Reply:
If I'm right freetube has no login support at all.